assets/javascripts/modules/control.js

```diff
@@ -0,0 +1,51 @@
+require('jquery');
+
+/*
+Helper to enable elements to control the value of an input else where on the page specified by a data attribute
+
+Control html markup:
+
+<label class="block-label block-label--inline" for="uk-phone-number-toggle-close">Yes
+    <input type="radio"
+           id="uk-phone-number-toggle-close"
+           class="js-toggle-trigger js-control"
+           value="yes"
+           name="uk-number"
+           required
+           data-control-target="countryCode"
+           data-control-value="44"/>
+</label>
+ */
+var $controlElems;
+
+var setup = function () {
+  $controlElems = $('.js-control');
+};
+
+var controlEvent = function ($controlElem) {
+  var $controlTarget = $('#' + $controlElem.data('controlTarget'));
+  var $controlValue = $controlElem.data('controlValue');
```

Comment:
more of a preference - had this conversation with @stingrayfunk.
I used to do this but it becomes quite confusing with large lists of vars. Minification and Uglify chomp through this duplication